tools/install_local_wheel.py

```diff
@@ -27,7 +27,7 @@ def configure_logging(lvl=logging.WARNING, logfile=None):
 def main():
```

Comment:
I remember talking last week about installing from wheel for nifake, and from source files for other drivers. Is that not what we're going with?

Reply:
We never install the wheel for nifake since we do not run system tests on it.

We do install the locally built nitclk wheel since that needs to be there to satisfy the dependencies when running nifake unit tests.

In the system tests, we build and install the wheel for each driver (and nitclk if the driver depends on it).

Reply:
Got it. Thanks.